### DISCUSSION TITLE
Live minimap when moving camera?

### DISCUSSION THREAD

**Discussion by shak2** [godotengine/godot-proposals#9905] [Editor]
A live small minimap when moving the camera in 3D please!

I know there is a preview tick box, but you can't move the camera using the arrows and rotate rings when in this preview mode.

A live little view from the camera when it's being moved around would be awesome.

I think Unreal engine has something like this.

Right now it's just trial and error tweaking back and forth over and over, mouse to keyboard to mouse to keyboard till I get the perfect perspective shot aligned.

Thanks!

**Comment by AThousandShips** [comment]
Are you meaning a preview or a minimap? They're very different concepts

You also have this proposal:
* https://github.com/godotengine/godot-proposals/issues/1017

Which should cover everything of this if you mean a preview by "minimap"

**Reply by shak2** [reply]
Yes, I meant a tiny preview window, that proposal is what I wanted, thanks.
